master/buildbot/reporters/stash.py

```diff
@@ -73,8 +84,8 @@ def send(self, build):
                 payload['description'] = yield props.render(description)
             if self.statusName:
                 payload['name'] = yield props.render(self.statusName)
-            response = yield self._http.post('/rest/build-status/1.0/commits/' + sha,
-                                             json=payload)
+            response = yield self._http.post(STASH_STATUS_API_URL
+                                                .format(sha=sha), json=payload)
```

Comment:
This line-break is inconsistent with the `STASH_COMMENT_API_URL` below (lines 130,131). The other layout is easier to read: 
`response = yield self._http.post(`
`....STASH_STATUS_API_URL.format(sha=sha), json=payload)`